mdpow/workflows/base.py

```diff
@@ -0,0 +1,180 @@
+# MDPOW: base.py
+# 2022 Cade Duckworth
+
+"""
+:mod:`mdpow.workflows.base` --- Automated workflow base functions
+=================================================================
+
+To analyze multiple MDPOW projects, provide :func:`project_paths`
+with the top-level directory containing all MDPOW projects' simulation data
+to obtain a :class:`pandas.DataFrame` containing the project information
+and paths. Then, :func:`automated_project_analysis` takes as input the
+aforementioned :class:`pandas.DataFrame` and runs the specified
+:class:`~mdpow.analysis.ensemble.EnsembleAnalysis` for all MDPOW projects
+under the top-level directory provided to :func:`project_paths`.
+
+Currently supported workflows - :mod:`~mdpow.workflows.dihedrals`
+
+.. seealso:: :mod:`~mdpow.workflows.registry`
+
+.. autofunction:: project_paths
+.. autofunction:: automated_project_analysis
+
+"""
+
+import os
+import re
+import pandas as pd
+
+from mdpow.workflows import registry
+
+import logging
+
+logger = logging.getLogger('mdpow.workflows.base')
+
+def project_paths(parent_directory=None, csv=None, csv_save_dir=None):
+    """Takes a top directory containing MDPOW projects and determines
+       the molname, resname, and path, of each MDPOW project within.
+
+       Optionally takes a .csv file containing `molname`, `resname`, and
+       `paths`, in that order. 
+
+       :keywords:
+
+       *parent_directory*
+           the path for the location of the top directory 
+           under which the subdirectories of MDPOW simulation
+           data exist, additionally creates a 'project_paths.csv' file
+           for user manipulation of metadata and for future reference
+
+       *csv*
+           .csv file containing the molecule names, resnames,
+           and paths, in that order, for the MDPOW simulation
+           data to be iterated over must contain header of the
+           form: `molecule,resname,path`
+
+       *csv_save_dir*
+           optionally provided directory to save .csv file, otherwise,
+           data will be saved in current working directory
+
+       :returns:
+
+       *project_paths*
+           :class:`pandas.DataFrame` containing MDPOW project metadata
+
+       .. rubric:: Example
+       
+       Typical Workflow::
+
+           project_paths = project_paths(parent_directory='/foo/bar/MDPOW_projects')
+           automated_project_analysis(project_paths)
+
+           'or'
+
+           project_paths = project_paths(csv='/foo/bar/MDPOW.csv')
+           automated_project_analysis(project_paths)
+
```

Comment:
If you look at https://mdpow--229.org.readthedocs.build/en/229/workflows/base.html#mdpow.workflows.base.project_paths
![image](https://user-images.githubusercontent.com/237980/228389825-8d075ef0-abfa-4458-89e1-d496bf53e935.png)
you see that the "or" is still formatted as code. 

The formatting below should fix it:

```suggestion
       Typical Workflow::
           project_paths = project_paths(parent_directory='/foo/bar/MDPOW_projects')
           automated_project_analysis(project_paths)
           
        or::
        
           project_paths = project_paths(csv='/foo/bar/MDPOW.csv')
           automated_project_analysis(project_paths)
           
```